src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -79,8 +79,8 @@
                 <attribute name="openapi_context">
                     <attribute name="summary">Address cart to given location, logged in Customer does not have to provide an email</attribute>
                 </attribute>
-                <attribute name="method">PATCH</attribute>
-                <attribute name="path">/orders/{id}/address</attribute>
+                <attribute name="method">PUT</attribute>
+                <attribute name="path">/orders/{id}</attribute>
```

Comment:
I'm not thrilled this solution, All checkout endpoints are individual and this changes is non consistent for others, It looks like `PUT` is for all order, not only for address

Reply:
I wonder if it wouldn't be better to leave `/address` in the path, it would be clearer for users and still consistent with other endpoints